docs/src/user-guide/getting-started.md

```diff
@@ -35,6 +66,10 @@ npx eslint yourfile.js
 # or
 
 yarn run eslint yourfile.js
+
+# or
+
+pnpm eslint yourfile.js
```

Comment:
```suggestion
```

Reply:
Do we need to remove the yarn command here as well?